python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -99,7 +99,7 @@ class RandomForestClassifier:
                         The minimum number of samples (rows) needed
                         to split a node.
     quantile_per_tree : boolean (default = False)
-                        Whether quantile is computed for individal trees in RF.
+                        Whether quantile is computed for individual trees in RF.
```

Comment:
Ran flake8 tests locally, this one was the only issue: 
```
./python/cuml/dask/ensemble/randomforestclassifier.py:102:80: E501 line too long (80 > 79 characters)
```
line is 1 character too long :P

Reply:
@dantegd , lol, you're kidding me...can we rephrase and say: 

"Whether quantile is computed for individual RF trees."

If yes, i'll fix and let you know to redo the flake8 tests.